tests/hardhat/Swap/swapTest.ts

```diff
@@ -438,5 +451,56 @@ describe("Swap Contract", () => {
           }),
       );
     });
+
+    it("Exact tokens -> BNB", async () => {
+      const deadline = await getValidDeadline();
+      // eslint-disable-next-line @typescript-eslint/no-floating-promises
+      await expect(
```

Comment:
What are we expecting here? I would add a test to really check that the debt has been repaid in the market. Not only for this case, but for every case (repay and supply)

Reply:
I shared same concern with Simeon and Kirill as Simeon was working on it earlier. He said we are just doing UT here and everything else will be covered in the mainnet testing.